app/assets/javascripts/surveys.js

```diff
@@ -3,41 +3,55 @@
+  var takeSurveyLink = function (text, className) {
+    className = className ? 'class="' + className + '"' : ''
+    return '<a ' + className + ' href="javascript:void()" id="take-survey" target="_blank" rel="noopener noreferrer">' + text + '</a>'
+  }
 
   var URL_SURVEY_TEMPLATE = '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
                             '  <div class="wrapper">' +
                                  cancelButton +
-                            '    <h1 id="feedback-heading">Tell us what you think of GOV.UK</h1>' +
-                            '    <p><a href="javascript:void()" id="take-survey" target="_blank" rel="noopener noreferrer">Take the 3 minute survey</a> This will open a short survey on another website</p>' +
+                                 surveyTitle +
+                            '    <p>' +
+                                 takeSurveyLink('Take the 3 minute survey', 'survey-primary-link') +
+                            '    This will open a short survey on another website</p>' +
                             '  </div>' +
                             '</section>',
     EMAIL_SURVEY_TEMPLATE = '<section id="user-satisfaction-survey" class="visible" aria-hidden="false">' +
                             '  <div class="wrapper">' +
                                  cancelButton +
                             '    <div class="inner-wrapper">' +
-                            '      <h1 id="feedback-heading">We\'d like to hear from you</h1>' +
+                                   surveyTitle +
                             '      <div id="email-survey-pre">' +
-                            '        <p><a href="#email-survey-form" id="email-survey-open" rel="noopener noreferrer" role="button" aria-expanded="false">Your feedback will help us improve this website</a></p>' +
+                            '        <a class="survey-primary-link" href="#email-survey-form" aria-labelledby="survey-title email-survey-open" id="email-survey-open" rel="noopener noreferrer" role="button" aria-expanded="false">' +
+                            '          Take a short survey to give us your feedback' +
+                            '        </a>' +
                             '      </div>' +
                             '      <form id="email-survey-form" action="/contact/govuk/email-survey-signup" method="post" class="js-hidden" aria-hidden="true">' +
                             '        <div id="feedback-prototype-form">' +
-                            '          <label for="survey-email-address">Tell us your email address and we\'ll send you a link to a quick feedback form.</label>' +
+                            '          <div id="survey-form-description" class="survey-form-description">We\'ll send you a link to a feedback form. It only takes 2 minutes to fill in.<br> Don\'t worry: we won\'t send you spam or share your email address with anyone.</div>' +
```

Comment:
We should keep the heading consistent between views so that users know what form they are  interacting with, with the prototype all context is lost so you do not know what your email is being used for.

I've ran this past @miaallers and she's 👍